src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -664,6 +665,7 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
             // append CodeFixCollection for each CodeFixProvider
             foreach (var provider in lazyConfigurationProviders.Value)
             {
+                using (Logger.LogBlock(FunctionId.CodeFixes_GetCodeFixesAsync, KeyValueLogMessage.Create(LogType.UserAction, m => CreateLogProperties(m, provider)), cancellationToken))
                 using (RoslynEventSource.LogInformationalBlock(FunctionId.CodeFixes_GetCodeFixesAsync, provider, cancellationToken))
                 {
                     var codeFixCollection = await TryGetFixesOrConfigurationsAsync(
```

Comment:
TryGetFixesOrConfigurationsAsync is called in multiple places.  i don't have confidence that we're properly measuring costs here.

Reply:
I think they are called in two places and all the events are added?